Kitodo/src/main/java/org/kitodo/production/filters/FilterMenu.java

```diff
@@ -305,6 +310,14 @@ public List<ParsedFilter> getParsedFilters() {
      * @return value of filterInEditMode
      */
     public String getFilterInEditMode() {
+        Map<String, String> parameters = FacesContext.getCurrentInstance().getExternalContext()
+                .getRequestParameterMap();
+        String input = parameters.get("input");
+        if (StringUtils.isNotBlank(input) && StringUtils.isBlank(filterInEditMode)) {
+            filterInEditMode = input;
+            parsedFilters.clear();
+            submitFilters();
+        }
```

Comment:
Maybe related to the problems when searching for projects. Should this not just return `filterInEditMode` as done in Master? The UI handling should be the same, independent of the used Search Backend. With this code on place one cannot even search first for a project and then teh second time, additionallfor a task state. The first filter gets removed internally.